CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.35.0 (2020-09-28)
+
+- Updated omnibar iframes to send a `hostid` to ensure messages are handled appropriately per iframe. [#164](https://github.com/blackbaud/auth-client/pull/164)
```

Comment:
```suggestion
- Updated omnibar iframes to provide `hostid` to ensure messages are handled appropriately per iframe. [#164](https://github.com/blackbaud/auth-client/pull/164)

Just trying to avoid the switch from plural `iframes` to the singular `a hostid`.


Also, is `iframes` the right capitalization? I think so, but I looked in the previous changelog items. and we had 1 `iframe` and 2 `IFRAME`.

Reply:
Updated to "provide" but left the capitalization.